Source/Infrastructure/Read/IReadModule.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Infrastructure.Read
+{
+    /// <summary>
+    /// In startup, create an override of public void ConfigureServicesCustom(IServiceCollection services).
+    /// Register IReadModule with s => new ReadModule(AppDomain.CurrentDomain).
+    /// Then instantiate the service with services.BuildServiceProvider().GetService<IReadModule></IReadModule>
+    ///     ();
+    /// </summary>
+    public interface IReadModule
+    {
+        void RegisterReadModelClassMaps(AppDomain appDomain);
+    }
```

Comment:
In Startup.Cs in each solution's Web project we instantiate this with, comments @einari ?

public override void ConfigureServicesCustom(IServiceCollection services)
        {
            services.AddSingleton<IReadModule>(s => new ReadModule(AppDomain.CurrentDomain));
            services.BuildServiceProvider().GetService<IReadModule>();
        }